core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -281,6 +281,8 @@ var ExplorationEditorPage = function() {
 
   this.navigateToMainTab = async function() {
     await action.waitForAutosave();
+    await waitFor.elementToBeClickable(
+      navigateToMainTabButton, 'Could not click navigate to Main Tab Button');
```

Comment:
`action.click` already waits for the element it's passed to be clickable, so I think this is redundant

Reply:
done